libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -962,6 +962,7 @@ void AC_PosControl::update_z_controller()
     float pos_target_zf = _pos_target.z;
 
     _vel_target.z = _p_pos_z.update_all(pos_target_zf, curr_alt, _limit.pos_down, _limit.pos_up);
+    _vel_target.z *= AP::ahrs().getEkfControlScaleZ();
```

Comment:
I would like to see the variable ekfNavVelGainScaler changed to be consistent with this variable ie getEkfControlScaleXY.

Reply:
done